modules/effects/arrow-effects-reactor/src/main/kotlin/arrow/effects/MonoK.kt

```diff
@@ -58,7 +68,7 @@ data class MonoK<A>(val mono: Mono<A>) : MonoKOf<A>, MonoKKindedJ<A> {
 
     fun <A> async(fa: Proc<A>): MonoK<A> =
       Mono.create { emitter: MonoSink<A> ->
-        fa { either: Either<Throwable, A> ->
+        fa(IOConnection()) { either: Either<Throwable, A> ->
```

Comment:
Same comment about `emitter` and `IOConnection`

Reply:
`IOConnection` is not exposed through Async anymore, mimicking cats.